packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/RequirementSubForm/index.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import { CWIconButton } from 'views/components/component_kit/cw_icon_button';
+import { CWSelectList } from 'views/components/component_kit/new_designs/CWSelectList';
+import { CWTextInput } from 'views/components/component_kit/new_designs/CWTextInput';
+import {
+  AMOUNT_CONDITIONS,
+  BLOCKCHAINS,
+  SPECIFICATIONS,
+  TOKENS,
+} from '../constants';
```

Comment:
I added these constants in this PR, but seems like they are also used in some other places but hardcoded. 

I think it a good idea to move this to a global constant, but maybe that's something for later.

cc: @masvelio